lib/online_order.rb

```diff
@@ -0,0 +1,79 @@
+require 'pry'
+require 'csv'
+require_relative './order.rb'
+require_relative './customer.rb'
+
+module Grocery
+
+  class OnlineOrder < Grocery::Order
+    attr_reader :id, :customer_id, :customer
+    attr_accessor :fill_status, :prducts
+
+    ONLINE_ORDERS = "support/online_orders.csv"
+
+    def initialize(order_id, products, cust_id, fill_status)
+      @id = order_id.to_i
+      @products = products
+      @customer_id = cust_id.to_i
```

Comment:
Instead of saving `@id` and `@products` yourself here, you should call `super(order_id, products)` and have `Order` do it for you.